bedrock/etc/templates/etc/firefox/retention/thank-you-a.html

```diff
@@ -25,41 +27,45 @@
   <canvas id="confetti-canvas"></canvas>
   <header>
     <h1>
-      <span class="white">It's all</span></br>
+    {% if LANG.startswith('en') %}
+      <span class="white">It’s all</span><br/>
       thanks to you
+    {% else %}
+      {{ _('It’s all thanks to you') }}
+    {% endif %}
     </h1>
     <p>
-      Choosing Firefox helps Mozilla make the Internet<br/>a better place. Here's what you can do next:
+      {{ _('Choosing Firefox helps Mozilla make the Internet a better place. Here’s what you can do next:') }}
     </p>
   </header>
   <section>
     <article>
-      <h1>Make Firefox your default browser</h1>
-      <div class="time">1 min action</div>
-        <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">Set as your default</a>
+      <h1>{{ _('Make Firefox your default browser') }}</h1>
+      <div class="time">{{ _('1 min action') }}</div>
+        <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">{{ _('Set as your default') }}</a>
     </article>
     <article>
-      <h1>Get Firefox on your phone</h1>
-      <div class="time">2 min install</div>
-      <a href="https://support.mozilla.org/kb/install-firefox-android-device-using-google-play">Download Firefox</a>
+      <h1>{{ _('Get Firefox on your phone') }}</h1>
+      <div class="time">{{ _('2 min install') }}</div>
+      <a href="https://support.mozilla.org/kb/install-firefox-android-device-using-google-play">{{ _('Download Firefox') }}</a>
     </article>
     <article>
-      <h1>Tell your friends about Firefox</h1>
-      <div class="time">1 min share</div>
-      <a href="https://twitter.com/home?status=%F0%9F%94%A5%2B%F0%9F%A6%8A=%F0%9F%99%8C%F0%9F%8F%BE%20Join%20me%20in%20the%20fight%20for%20an%20open%20web%20by%20choosing%20Firefox!%20https%3A//mozilla.org/firefox/new">Send a tweet</a>
+      <h1>{{ _('Tell your friends about Firefox') }}</h1>
+      <div class="time">{{ _('1 min share') }}</div>
+      <a href="{{ 'https://www.twitter.com/intent/tweet?url=%s&text=%s%s'|format('https://mozilla.org/firefox/new'|urlencode, '%F0%9F%94%A5%2B%F0%9F%A6%8A=%F0%9F%99%8C%F0%9F%8F%BE%20', _('Join me in the fight for an open web by choosing Firefox!')|urlencode)|e }}">{{ _('Send a tweet') }}</a>
```

Comment:
I copied the approach from existing pages, not sure if it's the best way to do it.

Reply:
Seems to work for me 👍